src/panels/config/integrations/integration-panels/ozw/ozw-node-dashboard.ts

```diff
@@ -0,0 +1,208 @@
+        .narrow=${this.narrow}
+        .route=${this.route}
+        .tabs=${ozwNetworkTabs(this.ozwInstance)}
+      >
+        <ha-config-section .narrow=${this.narrow} .isWide=${this.isWide}>
+          <div slot="header">
+            Node Management
+          </div>
+
+          <div slot="introduction">
+            View the status of a node and manage its configuration.
+          </div>
+
+          ${this._node
+            ? html`
+                <ha-card class="content">
+                  <div class="card-content">
+                    <b
+                      >${this._node.node_manufacturer_name}
+                      ${this._node.node_product_name}</b
+                    ><br />
+                    Node ID: ${this._node.node_id}<br />
+                    Query Stage: ${this._node.node_query_stage}
+                    ${this._metadata?.metadata.ProductManualURL
+                      ? html` <a
+                          href="${this._metadata.metadata.ProductManualURL}"
+                        >
+                          <p>Product Manual</p>
+                        </a>`
+                      : ``}
+                  </div>
+                  <div class="card-actions">
+                    <mwc-button @click=${this._refreshNodeClicked}>
+                      Refresh Node
+                    </mwc-button>
+                  </div>
+                </ha-card>
+              `
+            : ``}
+          ${this._metadata
+            ? html`
+                <ha-card class="content" header="Description">
+                  <div class="card-content">
+                    ${this._metadata.metadata.Description}
+                  </div>
+                </ha-card>
+                <ha-card class="content" header="Inclusion">
+                  <div class="card-content">
+                    ${this._metadata.metadata.InclusionHelp}
+                  </div>
+                </ha-card>
+                <ha-card class="content" header="Exclusion">
+                  <div class="card-content">
+                    ${this._metadata.metadata.ExclusionHelp}
+                  </div>
+                </ha-card>
+                <ha-card class="content" header="Reset">
+                  <div class="card-content">
+                    ${this._metadata.metadata.ResetHelp}
+                  </div>
+                </ha-card>
+                <ha-card class="content" header="WakeUp">
+                  <div class="card-content">
+                    ${this._metadata.metadata.WakeupHelp}
+                  </div>
+                </ha-card>
+              `
+            : ``}
+        </ha-config-section>
+      </hass-tabs-subpage>
+    `;
+  }
+
+  private async _fetchData() {
+    this._node = await fetchOZWNodeStatus(
+      this.hass!,
+      this.ozwInstance,
+      this.nodeId
+    );
+    this._metadata = await fetchOZWNodeMetadata(
+      this.hass!,
+      this.ozwInstance,
+      this.nodeId
+    );
```

Comment:
What will happen when we request a node that no longer/not exists?

Reply:
So this will not error but return None?

Reply:
Updated to catch a not_found error and show the error page. I used a separate _not_found property to prevent a quick flash of the not found page on initial load before it finishes fetching the node data initially.